frontend/model/state.js

```diff
@@ -345,11 +345,11 @@ const getters = {
         for (const toUser in paymentsFrom[ourUsername]) {
           for (const paymentHash of paymentsFrom[ourUsername][toUser]) {
             const { data, meta } = allPayments[paymentHash]
-            payments.push({ hash: paymentHash, data, meta, amount: data.amount, username: toUser })
+            payments.push({ hash: paymentHash, data, meta, amount: data.amount })
           }
         }
       }
-      return payments
+      return payments.sort((paymentA, paymentB) => paymentA.meta.createdDate < paymentB.meta.createdDate ? 1 : -1)
```

Comment:
Anywhere we're comparing ISOTimestamps, we should use `compareISOTimestamps`

Reply:
Okay.